### PR TITLE
Fix build by setting a well-known version for io-functions-commons definitions

### PR DESCRIPTION
**Pull request by gquadrati**
Since _[io-functions-commons/definitions.yaml](https://github.com/pagopa/io-functions-commons/blob/v18.0.7/openapi/definitions.yaml)_  has changed starting from version 19, this projects wasn't able to get builded anymore.

It's very urgent to fix it by setting a well-known version instead of relying on master one.

This problem is also affecting io-functions-service since is using this api, as visible [here](https://dev.azure.com/pagopa-io/io-functions-services/_build/results?buildId=11190&view=logs&j=976c91a6-2764-5391-e3d8-1993e9979d4d&t=976c91a6-2764-5391-e3d8-1993e9979d4d)

### CODE REVIEW

**Comment by balanza** [comment]
It's a good practice to reference a specific version, so well done!
As we're editing the file, would you mind to change `teamdigitale` with `pagopa`, too?

Thanks

**Comment by gquadrati** [comment]
> It's a good practice to reference a specific version, so well done!
> As we're editing the file, would you mind to change `teamdigitale` with `pagopa`, too?
> 
> Thanks

Done.. I also copied your PR and moved to new codegen (and to new version of openapi spec)

File: package.json [hunk 1]
```diff
@@ -21,14 +21,15 @@
     "dist:zip": "zip -r dist.zip . --exclude @.funcignore --exclude .funcignore",
     "predeploy": "npm-run-all generate build dist:modules",
     "generate:definitions": "rimraf ./generated/definitions && shx mkdir -p ./generated/definitions && gen-api-models --api-spec ./openapi/index.yaml --out-dir ./generated/definitions",
-    "generate:session-api": "rimraf ./generated/session-api && shx mkdir -p ./generated/session-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-backend/master/api_session.yaml --out-dir ./generated/session-api --request-types --response-decoders",
+    "generate:session-api": "rimraf ./generated/session-api && shx mkdir -p ./generated/session-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.15.3/api_session.yaml --out-dir ./generated/session-api --request-types --response-decoders",
```
Comment (balanza) [inline package.json:24]:
```suggestion
    "generate:session-api": "rimraf ./generated/session-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v7.15.3/api_session.yaml --out-dir ./generated/session-api --request-types --response-decoders",
```

**Review by balanza — APPROVED** [review]
Well done!

**Pull request merged**